lib/network_utils.ml

```diff
@@ -748,7 +747,7 @@ module Dhclient = struct
     let ipv6' = if ipv6 then "6" else "" in
     Filename.concat "/var/lib/xcp" (Printf.sprintf "dhclient%s-%s.conf" ipv6' interface)
 
-  let generate_conf ?(ipv6=false) interface options =
+  let[@warning "-27"] generate_conf ?(ipv6=false) interface options =
```

Comment:
I assume `ipv6` was the unused one here, I don't really have a nice solution for that, so lets keep it as is.